src/NumericalAlgorithms/Spectral/Spectral.cpp

```diff
@@ -321,6 +324,118 @@ const Matrix& linear_filter_matrix(const size_t num_points) noexcept {
       LinearFilterMatrixGenerator<BasisType, QuadratureType>>(num_points);
 }
 
+// Synthesize the corresponding functions that take a `Mesh` instead of template
```

Comment:
I don't understand what this is commenting on, and maybe because of that I'm not certain I understand what it's trying to tell me